scripts/utils.sh

```diff
@@ -35,4 +35,12 @@ function convertToLower() {
     else
         echo $1 | sed -e 's/\(.*\)/\L\1/'
     fi
+}
+
+function setGO_VERSION() {
+    export GO_VERSION=$(getGO_VERSION)
+}
+
+function getGO_VERSION() {
+    echo $(grep -m1 go go.mod | cut -d ' ' -f2).$(grep -m1 GO_PATCH_VERSION go.mod | cut -d ' ' -f3)
```

Comment:
Naively, this seems more suited to `versions.sh`. Admittedly, this is a nit, but is there a reason why it should be here instead of there? And have all of the callers of `setGO_VERSION` just `source versions.sh` instead?

Reply:
Admittedly I didn't know `versions.sh` existed when I started this PR. I've moved it now :)